.github/workflows/ci.yaml

```diff
@@ -1,73 +1,66 @@
 name: ci
 on:
   push:
-    branches: [main]
+    branches:
+      - main
   pull_request:
-    branches: [main]
-  schedule:
-    - cron: '15 22 * * *'
-  workflow_dispatch: {} # support manual runs
```

Comment:
Please keep the `workflow_dispatch` setting - it's very convenient to be able to trigger CI runs without pushing new commits.

Reply:
done